src/app/components/logIn/Login.tsx

```
@@ -0,0 +1,104 @@
+'use client';
+import React, { useEffect, useRef, useState } from 'react';
```

ESLint

[error] 'useEffect' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)